src/app/component/NewsletterSubscription/NewsletterSubscription.style.scss

```diff
@@ -0,0 +1,94 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+.NewsletterSubscription {
+    --newsletter-subscription-input-background: var(--input-background-color);
+    --newsletter-subscription-input-color: #{$black};
+    --newsletter-subscription-placeholder-color: #9f9c9c;
```

Comment:
What is this magic color? Where did it came from, use application-wise color pallet please!

Reply:
Done